docs/variants/novacustom_nv4x_adl/releases_heads.md

```diff
@@ -1,18 +1,96 @@
 # NovaCustom NV4x 12th Gen Dasharo (coreboot + heads) Release Notes
 
-This is a Dasharo Pro Package Release. To obtain access to the pre-built
-binaries you will have to
-[become the Dasharo Pro Package subscriber](../../ways-you-can-help-us.md#become-a-dasharo-pro-package-subscriber).
-You can do this by purchasing a Dasharo Pro Package product from our [shop](https://shop.3mdeb.com/shop/dasharo-pro-package/dasharo-corebootuefi-entry-subscription-upgrade-to-corebootheads-for-laptop-users/).
+This is a Dasharo Pro Package Release. To access the pre-built binaries,
+you need to [subscribe to the Dasharo Pro Package subscriber](https://docs.dasharo.com/ways-you-can-help-us/#become-a-dasharo-pro-package-subscriber).
+You can do this by purchasing a Dasharo Pro Package product from our [shop](https://shop.3mdeb.com/shop/dasharo-pro-package/1-year-dasharo-entry-subscription-for-network-appliance/).
+
 As a subscriber, you will receive access to all firmware updates for the
 duration of your subscription via the Dasharo Pro Package newsletter,
-and gain entry to the Dasharo Premier Support invite-only live chat
-on the Matrix network, enabling direct engagement with the Dasharo Team
-and fellow subscribers for personalized, priority assistance.
+and gain entry to the Dasharo Premier Support invite-only live chat on
+the Matrix network, enabling direct engagement with the Dasharo Team and fellow
+subscribers for personalized, priority assistance.
 
 Following Release Notes describe status of development of Dasharo (coreboot +
 Heads) firmware for NovaCustom NV4x 12th Gen.
 
+For details about our release process please read
+[Dasharo Standard Release Process](../../dev-proc/standard-release-process.md).
+
+{{ subscribe_form("310eac18-d302-478f-a617-5f5d65e8e0ac",
+"Subscribe to NovaCustom NV4x 12th Gen Dasharo Release Newsletter") }}
+
+## v0.9.2 - 2025-06-12
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1yWZ--zFPIsQhXZByf7nJIrasQYuRSf1yCi60lY_RGsQ/edit#gid=2042954457).
```

Comment:
it should be link to the osfv-results repo: https://github.com/Dasharo/osfv-results/blob/main/boards/NovaCustom/ADL_12th_Gen/NV41PZ/heads/v0.9.2-rc2-results.csv